gulpfile.js/tasks/clean.js

```diff
@@ -3,7 +3,7 @@ var del    = require('del')
 var config = require('../config')
 
 var cleanTask = function (cb) {
-  del([config.root.dest]).then(function (paths) {
+  del([config.root.dest + '/**', '!' + config.root.dest]).then(function (paths) {
```

Comment:
The issue I was having was that if you mount `public` as a Docker volume, you can't delete that folder as part of the clean task. My workaround here was to delete the _contents_ of public (hence the `/**`) but not the `public` directory itself. Does that make sense?